css/main.css

```diff
@@ -27,28 +27,100 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+li {
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.desktop-nav {
   display: none;
 }
 
-#logo {
-  font-family: "Roboto", sans-serif;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 18px;
-  line-height: 24px;
+.desktop-nav.visible {
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: #3c3a39;
+  min-height: 30vh;
+  z-index: 999;
+  width: 100%;
+}
+
+.desktop-nav ul {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+}
+
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.desktop-nav ul li {
+  border-bottom: 1px solid #6f6c6b;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  color: #fff5e1;
+  padding-left: 1rem;
+}
+
+.desktop-nav ul li a {
+  color: #fff5e1;
 }
 
 .menu-bars {
+  width: 18px;
+  font-size: 1.5rem;
+  color: white;
```

Comment:
Kindly change the font design following the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). ✌️